hdwallet/src/lib.rs

```diff
@@ -67,102 +81,54 @@ impl HDLattice {
         })
     }
 
-    pub fn master_key_from_seed(seed: &[u8; 64]) -> Result<[u8; 64], HDLatticeError> {
-        let mut hasher = hmac::Hmac::<Sha512>::new_from_slice(SALT).map_err(|_| {
-            HDLatticeError::KeyDerivationFailed("Failed to create HMAC".to_string())
-        })?;
-        hasher.update(seed);
-        Ok(hasher.finalize_fixed().into())
+    pub fn master_key_from_seed(seed: &[u8; 64]) -> Result<[u8; 32], HDLatticeError> {
```

Comment:
Is this a standard? To just use "m"?

So our initial master key is then also different - it's the same algo, but they're using a different salt. 

This means I can't use this to find our old keys - which is fine, I can just add this as a compatibility layer in the wallet.